index.html

```diff
@@ -332,7 +364,8 @@ <h3>skills</h3>
 
         <section class="contact-info">
```

Comment:
- [ ] [REQUIRED] Good job so far! Please remove `unneccery space` placed down of form inputs and give correct style to Contact Heading text(correct color =>`color: #262626;` and postion) 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/220963952-bfd35f13-45a6-4066-a324-f594561c1e7b.png)